### PR TITLE
Added alarms tab to cluster details as it was missing

### PR DESCRIPTION
**Pull request by jruddell**
image

image

### CODE REVIEW

File: src/app/plugins/kubernetes/components/alarms/AlarmDetailsLink.tsx [hunk 1]
```diff
@@ -19,11 +19,16 @@ const useStyles = makeStyles((theme: Theme) => ({
 const AlarmDetailsLink = ({ display, alarm }: Props) => {
   const [dialogOpened, setDialogOpened] = useState(false)
   const { link } = useStyles({})
-
+  const handleToggleDialogOpened = (e) => {
```
Comment (patil-pratik-87) [inline src/app/plugins/kubernetes/components/alarms/AlarmDetailsLink.tsx:22]: Maybe we can extend this to our tooltip too. I have seen the background registering a click when we try to click it.

**Review by patil-pratik-87 — APPROVED** [review]

File: src/app/plugins/kubernetes/components/alarms/AlarmDetailsLink.tsx (hunk 1 above)
Comment (jruddell) [reply src/app/plugins/kubernetes/components/alarms/AlarmDetailsLink.tsx:22]: 👍

File: src/app/core/elements/tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -19,6 +19,7 @@ import {
 import { middleRight } from 'core/elements/menu/defaults'
 import { styled } from '@material-ui/styles'
 import { debounce, CancellableDebounceFn } from 'utils/async'
+import { stopPropagation } from 'utils/fp'
```
Comment (patil-pratik-87) [inline src/app/core/elements/tooltip/Tooltip.tsx:22]: 😍

**Review by patil-pratik-87 — COMMENTED** [review]
Thanks for fixing the tooltip too 👍🏽

**Pull request merged**